### DISCUSSION TITLE
Not documented extension `disableFormField`

### DISCUSSION THREAD

**Discussion by renom** [ory/kratos#4169] [Q&A]
https://github.com/ory/kratos/blob/master/ui/node/attributes_input.go#L14
Is it okay to use not documented extension `disableFormField`?
Or it can be eventually removed from ory kratos codebase?

UPDATE: it seems it was already removed in [this commit](https://github.com/ory/kratos/commit/949e743ef9ddbc6e711f0174593f59f4fa3a1171)? (removed file `selfservice/flow/settings/extension.go`)

**Comment by vinckr** [comment]
What were you trying to solve?

**Reply by renom** [reply]
I'm trying to make a field read only in the GUI.

**Reply by vinckr** [reply]
I see, I think the best way to do this is directly in the frontend - you should have total control over it. 
Did you find a solution since you opened the discussion?

**Reply by renom** [reply]
I wanted to disable a field in `kratos-selfservice-ui-node`, so I didn't.
